spec/Records.tex

```diff
@@ -512,8 +512,8 @@ \subsection{The Method Receiver and the {\em this} Argument}
 \section{The {\em this} Method}
 \index{records!indexing}
 
-As with classes, records can be supplied with a \chpl{this} method.  This method
-defines the behavior of the indexing operator \chpl{[]}.
+As with classes, records can declare a \chpl{this} method.  Doing so
+permits record objects to be ``called''/''indexed.''
```

Comment:
I was just glancing at this out of curiosity, so not doing anything like a "review". But, should the first pair of tics on "indexed" actually be back-tics?